app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "divviup-app",
   "version": "0.1.0",
-  "private": true,
```

Comment:
I think we still want private: true here, since we do not want to accidentally publish this to npm